samples/chaincode/cc-tools-demo/README.md

```diff
@@ -0,0 +1,329 @@
+Here are the steps to accomplish this:
+
+* Clone the cc-tools-demo chaincode
+* Modify the chaincode to use FPC
+* Build your FPC CC-tools-demo chaincode
+* Launch a Fabric network
+* Install and instantiate your chaincode
+* Invoke transactions by using the FPC simple-cli
+
+## Prerequisites
+
+* This tutorial presumes that you have installed FPC as described in the FPC [README.md](../../../README.md#clone-fabric-private-chaincode) and `$FPC_PATH` is set accordingly.
+* We need a working FPC development environment. As described in the "Setup your Development Environment" Section of the FPC [README.md](../../../README.md#setup-your-development-environment), you can use our docker-based dev environment (Option 1) or setup your local development environment (Option 2).
+  We recommend using the docker-based development environment and continue this tutorial within the dev container terminal.
+* Moreover, within your FPC development you have already installed the FPC Go Chaincode Support components.
+  See the installation steps in [ecc_go/README.md](../../../ecc_go/README.md#installation).
+* We assume that you are familiar with Fabric chaincode development in go.
+  Most of the steps in this tutorial follow the normal Fabric chaincode development process, however, there are a few differences that we will highlight here.
+* Also, since the tutorial is on the integration between cc-tools and FPC, we expect you to have a grasp knowledge of [cc-tools](https://github.com/hyperledger-labs/cc-tools) framework and that you've at least tried to run the [cc-tools-demo](https://github.com/hyperledger-labs/cc-tools-demo) once by yourself on a Fabric network
+
+## Clone the cc-tools-demo chaincode
+
+We need to clone the chaincode folder from the [cc-tools-demo](https://github.com/hyperledger-labs/cc-tools-demo) repository here.
+
+**Note**: If you're not inside the dev environment, git commands will need sudo permissions like the following:
+
+```bash
+export ccToolsDemoPath=$FPC_PATH/samples/chaincode/cc-tools-demo
+git clone -n --no-checkout --depth=1 --filter=tree:0 https://github.com/hyperledger-labs/cc-tools-demo.git "$ccToolsDemoPath/chaincode"
+cd "$ccToolsDemoPath/chaincode" || { echo "$ccToolsDemoPath/chaincode does not exist" ; exit 1; }
+sudo chown -R $USER $ccToolsDemoPath
+git config --global --add safe.directory /src/github.com/hyperledger/fabric-private-chaincode/samples/chaincode/cc-tools-demo/chaincode
+git sparse-checkout set --no-cone chaincode/*
+git checkout
+mv chaincode/* $ccToolsDemoPath
+cd $ccToolsDemoPath
+sudo rm -r $ccToolsDemoPath/chaincode
+```
+
+**Note:** If you're inside the dev environment, you don't need the `sudo` permissions and don't need to `chown` the directory.
+The chaincode code structure is different than normal chaincode as it's using the cc-tools framework.
```

Comment:
I think you need to add an enter here ... I don't think that last sentence is part of the note, correct?

Reply:
You're right. Done